deployment/quick-start/services-configuration.yaml.template

```diff
@@ -58,6 +58,12 @@ cluster:
 #      description: Default VC.
 #      capacity: 100
 
+#Uncomment following lines if you want to customize hdfs
+#hadoop-data-node:
+#  # storage path for hdfs, support comma-delimited list of directories, eg. /path/to/folder1,/path/to/folder2 ...
+#  # if left empty, will use cluster.common.data-path/hdfs/data
+#  storage_path:
```

Comment:
what's the default value?

why not make it contains config default value and also give user flexibility to config multi path?
storage_path: /datastorage

Reply:
@YanjieGao 
Default value is `cluster_config[common][storage]/hdfs/data`, which might not be `/datastorage`. 
The logic is: if admin give  a specific hdfs storage, then use it, if not,  use global storage config. 
Ideally, if we allow introduce other services' config in `yaml`, then here we could set `$cluser_config.common.storage/hdfs/data`, which would be more clear

Reply:
For user only view this config file can't find cluster.common.data-path position. It will be confused for user. Because not all user know to query github code base to find cluster_config[common][storage] cluster object model config.

It is better to tell user clear the default path is where.

We should assume user maybe only get the context of current config file.

Reply:
Actually, I don't think user should see here, the default value is for advanced user or dev. In our design, user should overwrite this value in `services-configuration.yaml`, which contains the necessary context. 

If we introduce some hard-code path here(even only in comments not in code) which will couple this file with other service.

Reply:
I see, misunderstanding this as end user yaml file & my intent is not hard code (intend is to tell user could find this config default value is this file's data-path config). 

In my understand default value is for not advanced user and advanced user will know customized it.

Reply:
not big problem. Could continue